codes/models/alt_growth_rates/USA_adm1.do

```diff
@@ -32,6 +33,22 @@ drop if cum_confirmed_cases < 10
 bysort adm1_id: egen total_obs = total((adm1_id~=.))
 drop if total_obs < 4 // drop state if less than 4 obs
 
+// flag which admin unit has longest series
```

Comment:
comment again that it would be great to put this and the testing regime functionality into standalone functions/.ado files if/when we have a chance since it's copy/pasted across each country right now, I think?

Reply:
it's not exactly copy/paste b/c diff countries are at diff admin unit levels and some countries use cumulative confirmed cases v. active cases. could be in a function but would def be extra work.